### DISCUSSION TITLE
How to hide the download option?

### DISCUSSION THREAD

**Discussion by jrfullstack** [katspaugh/wavesurfer.js#3506] [Q&A]
Hello, 
I would like to hide the download option of the player, is there an option?
React - NextJS

![image](https://github.com/katspaugh/wavesurfer.js/assets/96630703/6659c8f5-75fc-4951-bd45-aefb04b05d1b)

THX

**Comment by katspaugh** [comment]
You can pass a custom audio element in the `media` option and add a controlslist=“nodownload” attribute to it.

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#controlslist
